### ISSUE TITLE
Apururu not responding for Windurst mission #2

### ISSUE THREAD

**Issue by Smeagle55** [EdenServer/community#2139]
### Checklist

- [x ] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [ x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [ x] I have provided my client version in the details. (type /ver into your game window)

### Details
I have finished charging all 6 Mana Orbs in the SE tower in East Sarutabaruta. I then proceeded to the main laboratory door in the SE Tower to receive the cutscene displaying the completion of charging all 6 orbs. The last thing I believe I am supposed to do is go back to Apururu (like she said before she got bugged out and is unresponsive) and talk to her upon finishing the job, but she is not unresponsive and says nothing. I have tried rezoning, restarting and also going back to the SE tower with no changes at all. The problem in short; I believe Apururu is bugged and will not respond to finish mission number two for Windurst.  *Version 30181205_0

**Comment by Althaia** [comment]
Hiya, I spoke with Smeagle about this a couple of days ago and together we discovered he skipped a step, specifically the part where you're supposed to go back and retrieve all six mana orbs. (Which itself is then followed by a Cutscene as you're exiting the ruins).

I just checked in with him again today and it looks like that was the only problem, he was able to finish the mission, so feel free to close this. ;D
